src/Transition/SupportFiles/Report Variables 9-2-0 to 9-3-0.csv

```diff
@@ -1,6 +1,5 @@
 9.2.0,9.3.0,Transition notes
 12,12,These numbers should be the number of report variables in the following list (including deletes). Two columns/numbers.
-old variable name,new variable name,-- add variable names (before this line) and leave off units -- <DELETE> to delete
 Water to Water Heat Pump Load Side Heat Transfer Rate,Heat Pump Load Side Heat Transfer Rate,
 Water to Water Heat Pump Load Side Heat Transfer Energy,EIR Heat Pump Load Side Heat Transfer Energy,

```

Comment:
@mitchute These same changes are also in #7448 but it's better that these get fixed in a separate PR for better visibility. After this merges, I'll resolved conflicts over there.

Note that this line also needs a fix, delete EIR. 
~EIR~ Heat Pump Load Side Heat Transfer Energy

Reply:
@mjwitte Thanks for pointing that out. I'll make the changes soon.